### PR TITLE
pwntools 4.9.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
```
    * pwntools contains conflicting version recursive dependencies:
        openssl@3, openssl@1.1
```

File: Formula/pwntools.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Pwntools < Formula
   end
 
   depends_on "rust" => :build
+  depends_on "capstone"
```
Comment (cho-m) [inline Formula/pwntools.rb:24]: I don't think this includes the python bindings

File: Formula/pwntools.rb [hunk 2]
```diff
@@ -160,28 +156,23 @@ class Pwntools < Formula
...
 
   resource "sortedcontainers" do
     url "https://files.pythonhosted.org/packages/e8/c4/ba2f8066cceb6f23394729afe52f3bf7adec04bf9ed2c820b39e19299111/sortedcontainers-2.4.0.tar.gz"
     sha256 "25caa5a06cc30b6b83d11423433f65d1f9d76c4c6a0c90e3379eaa43b9bfdb88"
   end
 
-  resource "unicorn" do
-    url "https://files.pythonhosted.org/packages/b3/9c/d6d3de658534bbc82fa73dea711860e345c5deec8974c5bd25588c399f9b/unicorn-2.0.1.tar.gz"
-    sha256 "62ad9bb3249ae2794b4cffe8d17dc0aef3f7db12a1938a7a63bccd4797bf4e30"
-  end
```
Comment (cho-m) [inline Formula/pwntools.rb:180]: Also still needed as `unicorn` formula is only the C library. Need this for python bindings.

File: Formula/pwntools.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/pwntools.rb]: I see

File: Formula/pwntools.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/pwntools.rb]: updated

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**